ext/DifferentiationInterfaceTapirExt/mutating.jl

```diff
@@ -0,0 +1,13 @@
+function DI.value_and_pullback!!(f!, y, dx, ::AutoTapir, x, dy, extras::Nothing)
+    rrule = build_rrule(f!, y, x)
```

Comment:
this is very inefficient, I'll put it in a preparation step once the debugging is done